libpod/container_internal.go

```diff
@@ -789,6 +847,26 @@ type resolvConf struct {
 	options       []string
 }
 
+// RootUID returns the root user mapping from container
+func (c *Container) RootUID() int {
```

Comment:
Throw this and RootGID() in `container.go` with our other public accessors

Reply:
Fixed